Directory.Build.props

```diff
@@ -20,7 +20,7 @@
 
   <!-- Set these properties early enough for libraries as they import the Arcade SDK not early enough.  -->
   <PropertyGroup Condition="'$(SkipImportArcadeSdkFromRoot)' == 'true'">
-    <RepoRoot>$(MSBuildThisFileDirectory)</RepoRoot>
+    <RepoRoot>$(MSBuildThisFileDirectory)\</RepoRoot>
```

Comment:
`$([MSBuild]::EnsureTrailingSlash('$(MSBuildThisFileDirectory)'))` to append the right type of slash?

Reply:
Didn't even know that function existed. Thanks!